test/transaction.test.js

```diff
@@ -25,10 +24,21 @@ test.beforeEach(t => {
  * @param t the test object
  * @param method
  */
-let statusSuccess = async (t, method, params) => {
-    let response = await t.context.web3data.transaction[method](TX_HASH)
+const statusSuccess = async (t, { method, params = {} }) => {
+    const response = await t.context.web3data.transaction[method]()
     t.is(response.status, 200)
 }
 statusSuccess.title = (providedTitle = '', input) =>  `Successfully calls ${input.method} and returns status of 200`
 
-test([statusSuccess], 'getGasPrediction')
+const returnsString = async (t, { method, params = {} }) => {
+    const response = await t.context.web3data.transaction[method]()
+    t.is(typeof response, 'string')
+    t.is(typeof parseInt(response), 'number')
+}
+returnsString.title = (providedTitle = '', input) => `Successfully calls ${input.method} and returns string value`
+
+test([statusSuccess], {method:'getGasPrediction'})
+test([returnsString], {method:'getGasPrice'})
```

Comment:
@taylorjdawson looks like a lot of abstraction for the repetitive things. Its nice, but in test files, the best format is the simplest for others to easily parse, update & maintain. That is the core purpose of tests, conforming to the majority standard, they shouldn't be refactored to the best, because likely that will actually cause more problems and not give accurate results. 
Rule of thumb: Keep your tests dumb and your code smart. :)

Reply:
Okay! I was trying to follow avajs [best practices](https://github.com/avajs/ava/blob/master/docs/01-writing-tests.md#reusing-test-logic-through-macros).